modules/nw-sriov-configuring-device.adoc

```diff
@@ -20,8 +20,13 @@ You can configure an SR-IOV network device by creating a SriovNetworkNodePolicy
 [NOTE]
 =====
 When applying the configuration specified in a `SriovNetworkNodePolicy` object, the SR-IOV Operator might drain the nodes, and in some cases, reboot nodes.
+
+Before applying a configuration update, ensure that the following conditions are met:
+
+* There are enough available nodes in your cluster to handle the evicted workload from drained nodes
```

Comment:
I get why you added this here because it's related to this note content, but these seem like actionable prerequisites. I'd consider adding them to the prereq section.

Reply:
@bergerhoffer, good idea!